openprescribing/templates/faq.html

```diff
@@ -71,6 +72,8 @@ <h3 id="cite">How do I cite OpenPrescribing?</h3>
 <h3 id="timeperiod">What time period does the dataset cover?</h3>
 <p>We have data from the past 5 years on OpenPrescribing.net, and from 1998 in the <a href="https://openprescribing.net/long_term_trends/">Long Term Trends section</a>.</p>
 
+<h3 id="compare">Why do OpenPrescribing figures not always match with other prescribing data analysis platforms?</h3>
+<p>At OpenPrescribing we anticipate that our results will not always be exactly the same as those displayed on alternative data analysis platforms. There are two common reasons for this. Firstly, different platforms use different underlying datasets. At OpenPrescribing we use two openly available datasets: the <a href="https://www.nhsbsa.nhs.uk/prescription-data/prescribing-data/english-prescribing-data-epd">English Prescribing dataset (EPD)</a> and the <a href="https://www.nhsbsa.nhs.uk/prescription-data/dispensing-data/prescription-cost-analysis-pca-data"> Prescription Cost Analysis (PCA)</a>. Secondly, different platforms will make different choices in their analytic code for preparing the data and displaying it to users.</p>
```

Comment:
I think at the very end you should include a line that says something like 

_All our code for data preparation, analysis and visualisation  is openly available on our [Bennett Institute GIthub](https://github.com/ebmdatalab/openprescribing) for inspection and re-use by anyone. Additionally for each measure you have the choice to "View technical details for this measure" to illustrate exactly what we are measuring in code_